Cross.toml

```diff
@@ -0,0 +1,9 @@
+[target.x86_64-unknown-linux-gnu]
+pre-build = [
+    "rm -rf /usr/local/go && curl -L https://go.dev/dl/go1.20.5.linux-amd64.tar.gz | tar -xz -C /usr/local && ln -s /usr/local/go/bin/go /usr/local/bin/ && go version"
```

Comment:
could we move these into the build steps / github action instead? Less places to keep track of build logic

Reply:
Especially since these cause side effects

Reply:
Alternatively, I don't know how exactly `Cross.toml` works, but could we have only one pre-build section, that applies to both/all archs?

Reply:
Done - see [a72cfdb](https://github.com/filecoin-station/rusty-lassie/pull/21/commits/a72cfdb2dadfca7ac07123621edafc74139fa141) and [9d87e57](https://github.com/filecoin-station/rusty-lassie/pull/21/commits/9d87e5739a5ea831fa58cfc59c4de7307c74a6ff)